threesums.c

```diff
@@ -0,0 +1,28 @@
+/*Jared Wasserman*/
```

Comment:
Perfect! Just again, add a program description at the top or comments throughout!